lib/gems/pending/appliance_console.rb

```diff
@@ -150,37 +150,44 @@ module ApplianceConsole
       clear_screen
       selection = ask_with_menu("Advanced Setting", AS_OPTIONS, nil, true)
       case selection
-      when I18n.t("advanced_settings.dhcp")
```

Comment:
I think we've lost the ability to i18n these lines.  I'm ok with this for now, but that should probably be put back.